doc/user-getting-started-docker-dotnet.md

```diff
@@ -115,53 +100,29 @@ myst-appbuilder Dockerfile
 `/home`, `/bin`, etc. It also contains our application under `/app`.
 The dotnet runtime is also included.
 
-## Package the application
-
-dotnet runtime requires more heap memory than Mystikos provides
-by default. To expand the memory, we need to sign or package the application
-with a configuration file, in which a higher limit of heap size can be
-specified as follows:
-```
-{
-    // OpenEnclave specific settings
-    "Debug": 1,
-    "StackMemSize": "400k",
-    "NumUserThreads": 8,
-    "ProductID": 1,
-    "SecurityVersion": 1,
-
-    // Mystikos specific settings
-    // The heap size of the kernel and user application. Increase this setting
-    // if your app experienced OOM.
-    "MemorySize": "512m",
-    // The path to the entry point application in rootfs
-    "ApplicationPath": "/usr/bin/dotnet",
-    // The parameters to the entry point application
-    "ApplicationParameters": ["/app/sum.dll"],
-    // Whether we allow "ApplicationParameters" to be overridden by command line options of "myst exec"
-    "HostApplicationParameters": true,
-}
-```
-You can ignore most of the settings for now except for `MemorySize`. 512 MB is the minimum required
-by dotnet runtime, and you can increase it as needed. For details of the config file, please refer to
-[signing and packaging](./sign-package.md)
+## Create a CPIO archive and run the program inside an SGX enclave
 
-Save the above config file to `config.json`, and package the app with:
+```bash
+myst mkcpio appdir rootfs
+myst exec-sgx --memory-size 512m rootfs /usr/bin/dotnet /app/sum.dll 1 2 3
 ```
-openssl genrsa -out private.pem -3 3072
-myst package appdir private.pem config.json
-```
-
-## Run the program inside a SGX enclave
 
-Packaging produces an executable under `$PWD/myst/dotnet`, which can be launched
-like any system-installed dotnet runtime (except the execution actually happens in a TEE):
-```
-myst/bin/dotnet /app/sum.dll 1 2 3
-```
+On the last command, we pass in `--memory-size 512m` to tell Mystikos to
```

Comment:
"myst exec-sgx" approach is only suitable for development and debug phase. "Packaged" mode is required for production. The doc should call out the intended usage of "must exec-sgx" mode and refer the reader to other doc for packaged mode details.

Reply:
That is covered in get started guide for C. I emphasized `myst exec-sgx` is debug mode and add a reference to packaging here.